src/cmd/linuxkit/pkglib/pkglib.go

```diff
@@ -90,6 +92,13 @@ func NewFromCLI(fs *flag.FlagSet, args ...string) (Pkg, error) {
 		return Pkg{}, err
 	}
 
+	if devMode {
+		pi.Org = os.Getenv("USER")
```

Comment:
This does not seem to honour the `--org` override but probably should. On some of my systems my user name is different to my hub name. Or is the intention to not push in `--dev` mode, in which case we might want to explicitly disable this

Reply:
ah, on all my systems my username is different to my hub name yeah...